src/Common/tests/Tests/Interop/cgroupsTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace Common.Tests
+{
+    public class cgroupsTests
+    {
+        [Theory]
+        [InlineData(true, "0",  0)]
+        [InlineData(false, "max",  0)]
+        [InlineData(true, "1k",  1024)]
+        [InlineData(true, "1K",  1024)]
+        public static void ValidateTryReadMemoryValue(bool expectedResult, string valueText, ulong expectedValue)
+        {
+            string path = Path.GetTempFileName();
+            try
+            {
+                File.WriteAllText(path, valueText);
+
+                var result = Interop.cgroups.TryReadMemoryValueFromFile(path, out ulong val);
+
+                Assert.Equal(expectedResult, result);
+                if (result)
+                {
+                    Assert.Equal(expectedValue, val);
+                }
+            }
+            finally {
```

Comment:
Style: Open brace should be on the next line (repeats 3x).

Reply:
Thanks. Fixed now.